src/Pos/Wallet/WalletMode.hs

```diff
@@ -59,7 +59,7 @@ import           Pos.WorkMode                (RawRealMode)
 deriving instance MonadBalances m => MonadBalances (WalletWebDB m)
 
 instance MonadIO m => MonadBalances (WalletDB m) where
-    getOwnUtxo addr = WS.getUtxo >>= return . filterUtxoByAddr addr
+    getOwnUtxo addr = (filterUtxoByAddr addr) <$> WS.getUtxo
```

Comment:
Redundant brackets around `filter...`. You probably need to apply hlint twice at least, because it's not recursive.